static/app/views/alerts/rules/metric/incompatibleAlertQuery.tsx

```diff
@@ -111,15 +110,6 @@ export function IncompatibleAlertQuery(props: IncompatibleAlertQueryProps) {
     return null;
   }
 
-  const eventTypeError = props.eventView.clone();
-  eventTypeError.query += ' event.type:error';
-  const eventTypeTransaction = props.eventView.clone();
-  eventTypeTransaction.query += ' event.type:transaction';
-  const eventTypeDefault = props.eventView.clone();
-  eventTypeDefault.query += ' event.type:default';
-  const eventTypeErrorDefault = props.eventView.clone();
-  eventTypeErrorDefault.query += ' event.type:error or event.type:default';
-
```

Comment:
Seems like this just wasn't used anywhere?

Reply:
Yeah, these values we're being used, there was a refactor that happened here and they likely got left behind